src/main/kotlin/com/unblu/demo/sample/jwt/MainController.kt

```diff
@@ -8,7 +8,7 @@ import org.springframework.web.bind.annotation.GetMapping
 @Controller
 class MainController(private val unbluConfiguration: UnbluConfiguration) {
 
-    @GetMapping
+    @GetMapping("index")
```

Comment:
is this really needed?

Reply:
At least in my test setup, with my rev proxy, it does not work without it. But to keep it as it was i'll remove.

Reply:
updated